src/sentry/sentry_apps/api/bases/sentryapps.py

```diff
@@ -135,7 +161,7 @@ def _get_organization_for_superuser_or_staff(
 
         if context is None:
             error_message = f"Organization '{organization_slug}' does not exist."
-            raise ValidationError({"organization": to_single_line_str(error_message)})
+            raise SentryAppError from ValidationError(to_single_line_str(error_message))
```

Comment:
So we use `from`/chain here instead of wrapping since we want to get the wrapped error in the error handler (using `__cause__`). 

Sidenote: I guess we could chain in other errors we'd just need to use .`__cause__ ` to get the underlying error when stringifying. I'm not sure how chaining vs wrapping errors affects the stack trace or other parts tho

Reply:
Ok I gave up this was leading to type errors that would not be easily fixed